### PR TITLE
Setup script

### PR DESCRIPTION
**Pull request by schoren**
This PR adds a setup script and related documentation, so it's easier for new users to get started. This command can be run as a single copy/paste without downloading anything.

## Changes

- add setup script
- add docs

## Checklist

- [x] tested locally
- [ ] added new dependencies
- [x] updated the docs
- [ ] added a test

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+NAMESPACE="tractest"
```
Comment (mathnogueira) [inline setup.sh:3]:
```suggestion
NAMESPACE="tracetest"
```

File: setup.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+SKIP_PMA="no"
+
+while [[ $# -gt 0 ]]; do
+  case $1 in
+    -n|--namespace)
+      EXTENSION="$2"
```
Comment (mathnogueira) [inline setup.sh:9]: Shouldn't it be `NAMESPACE="$2"?

Comment (schoren) [reply setup.sh:9]: yup

**Review by xoscar — APPROVED** [review]
LGTM

File: docs/installing.md [hunk 3]
```diff
@@ -1,62 +1,85 @@
...
+- [Kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+## **Installation**
+
+### Install script
+
+We provide a simple install script that can install all required components:
+
+```
+curl -L https://raw.githubusercontent.com/kubeshop/main/setup.sh | bash -s
```
Comment (schoren) [inline docs/installing.md:23]: this URL won't work until the PR is merged. You can try this command like this:

```
curl -L https://raw.githubusercontent.com/kubeshop/tracetest/sample-setup-script/setup.sh | bash -s
```

**Review by kdhamric — APPROVED** [review]
Looks good - this will help people get started quicker!

**Review by mathnogueira — CHANGES_REQUESTED** [review]
Installation fails on my machine when running on minikube :cry: 

image

**Comment by schoren** [comment]
@mathnogueira I've done a lof of changes, so now you should be able to see it working. Just to be safe, you can do a cleanup (replace ` my-custom-namespace` for whatever namespaces you've used):

```
helm delete -n demo demo
helm delete -n my-custom-namespace tracetest
kubectl delete crd certificaterequests.cert-manager.io certificates.cert-manager.io challenges.acme.cert-manager.io clusterissuers.cert-manager.io issuers.cert-manager.io orders.acme.cert-manager.io jaegers.jaegertracing.io cert-manager-webhook
kubectl delete validatingwebhookconfigurations jaeger-operator-validating-webhook-configuration
kubectl delete mutatingwebhookconfigurations jaeger-operator-mutating-webhook-configuration
kubectl delete clusterrolebinding "jaeger-operator-proxy-rolebinding" "manager-rolebinding"
kubectl delete clusterrole "jaeger-operator-metrics-reader" "manager-role" "proxy-role"
kubectl delete ns observability cert-manager demo my-custom-namespace
```

**Review by mathnogueira — APPROVED** [review]

**Review by kdhamric — APPROVED** [review]
Looks good!

File: docs/installing.md [hunk 4]
```diff
@@ -1,62 +1,90 @@
...
-The following command will uninstall Tracetest with Postgres:
-
-```sh
-# Delete releases
-
-helm delete tracetest
-```
+# Installation
+
+During the setup, we'll deploy Tracetest, and Postgres with Helm.
```
Comment (jfermi) [inline docs/installing.md:3]:
```suggestion
During the setup, we'll deploy Tracetest and Postgres with Helm.
```

File: docs/installing.md [hunk 5]
```diff
@@ -1,62 +1,90 @@
...
+
+For the architectural overview of the components please check the [Architecture](architecture.md) page.
```
Comment (jfermi) [inline docs/installing.md:5]:
```suggestion
For the architectural overview of the components, please check the [Architecture](architecture.md) page.
```

File: docs/installing.md [hunk 6]
```diff
@@ -1,62 +1,90 @@
...
+
+### Install script
+
+We provide a simple install script that can install all required components:
+
+```
+curl -L https://raw.githubusercontent.com/kubeshop/main/setup.sh | bash -s
+```
+
+This command will install tracetest using the default settings. You can configure the following options:
```
Comment (jfermi) [inline docs/installing.md:26]:
```suggestion
This command will install Tracetest using the default settings. You can configure the following options:
```

File: docs/installing.md [hunk 7]
```diff
@@ -1,62 +1,90 @@
...
+
+```
+curl -L https://raw.githubusercontent.com/kubeshop/main/setup.sh | bash -s -- --skip-pma --namespace my-custom-namespace
+```
+
+### Using Helm
+
+Container images are hosted on the Docker Hub [Tracetest repository](https://hub.docker.com/r/kubeshop/tracetest).
+
+Tracetest currently supports two traces backend: Jaeger and Grafana Tempo.
```
Comment (jfermi) [inline docs/installing.md:49]:
```suggestion
Tracetest currently supports two trace backends: Jaeger and Grafana Tempo.
```

File: docs/installing.md [hunk 8]
```diff
@@ -1,62 +1,90 @@
...
+
+#### **Jaeger**
+
+Tracetest uses [Jaeger Query Service `16685` port](https://www.jaegertracing.io/docs/1.32/deployment/#query-service--ui) to find Traces using gRPC protocol.
+
+The commands below will install Tracetest connecting to Jaeger tracing backend on `jaeger-query:16685`.
```
Comment (jfermi) [inline docs/installing.md:55]:
```suggestion
The commands below will install Tracetest connecting to the Jaeger tracing backend on `jaeger-query:16685`.
```

File: docs/installing.md [hunk 9]
```diff
@@ -1,62 +1,90 @@
...
+  --set tracingBackend=jaeger \
+  --set jaegerConnectionConfig.endpoint="jaeger-query:16685" # update this value to point to your jaeger install
+```
+
+#### **Grafana Tempo**
+
+Tracetest uses [Grafana Tempo's Server's `9095` port](https://grafana.com/docs/tempo/latest/configuration/#server) to find Traces using gRPC protocol.
+
+
+The commands below will install the Tracetest application connecting to Grafana Tempo tracing backend on `grafana-tempo:9095`:
```
Comment (jfermi) [inline docs/installing.md:72]:
```suggestion
The commands below will install the Tracetest application connecting to the Grafana Tempo tracing backend on `grafana-tempo:9095`:
```

**Review by jfermi — COMMENTED** [review]
Made a few editing suggestions.

**Pull request merged**